android/engine/src/main/res/values/strings.xml

```diff
@@ -12,6 +12,7 @@
     <string name="app_logo">Application logo</string>
     <string name="powered_by">Powered By</string>
     <string name="app_version">App version %1$d(%2$s)</string>
+    <string name="ig_version">ImplementationGuide version %s</string>
```

Comment:
how is the `ig_version` different from the `app_version`? Is the `app_version` taken from the composition referenced within the ig?

Reply:
If so, maybe we rename this and the line above to:

```suggestion
    <string name="app_version">App Composition version %1$d(%2$s)</string>
    <string name="ig_version">App ImplementationGuide version %s</string>
```

to make clear they're both defining the app

Reply:
the ig_version should be removed. It was initially added when we thought IG would have a different version number

Reply:
```suggestion
    <string name="app_version">App version %1$d(%2$s)</string>
```